src/pages/Organization/OrganizationFacilities.tsx

```
@@ -114,62 +114,59 @@ export default function OrganizationFacilities({
-                  </div>
-                  <CardContent className="p-6">
-                    <div className="flex flex-col h-full">
-                      <div className="flex items-start justify-between">
-                        <div>
-                          <h3 className="text-md font-medium text-gray-900">
-                            {facility.name}
-                          </h3>
-                          <div className="font-medium">
-                            {facility.facility_type}
-                          </div>
+              <Card className="h-full hover:border-primary/50 transition-colors overflow-hidden">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)